src/renderer/modules/webpack/index.ts

```diff
@@ -51,9 +81,12 @@ export let signalStart: () => void;
  * @internal
  * @hidden
  */
-export const waitForStart = new Promise<void>((resolve) => (signalStart = resolve));
+export const waitForStart = new Promise<void>((resolve) => {
+  signalStart = resolve;
+  webpack.signalStart = signalStart;
+});
 
-const sourceStrings: Record<number, string> = {};
+export const sourceStrings: Record<number, string> = {};
```

Comment:
Why does this need to be exported? It's meant to be private, for our own functions to be able to find modules by string.

Reply:
Not yet resolved